source/beamline/AMXspress3XRFDetector.cpp

```diff
@@ -239,9 +250,8 @@ void AMXspress3XRFDetector::onTriggerSourceTriggered(AMDetectorDefinitions::Read
 AMAction3 * AMXspress3XRFDetector::createInitializationAction()
 {
 	AMListAction3 *initializeAction = new AMListAction3(new AMListActionInfo3("Arm detector", "Arm detector"));
-	initializeAction->addSubAction(AMActionSupport::buildControlMoveAction(eraseControl_, 1.0));
+	initializeAction->addSubAction(createEraseAction());
 	initializeAction->addSubAction(AMActionSupport::buildControlMoveAction(updateControl_, 1.0));
-	initializeAction->addSubAction(AMActionSupport::buildControlMoveAction(eraseControl_, 0.0));
```

Comment:
In testing, the code as written seemed to work okay. I'll double check with David B. though.